__mocks__/vscode.js

```
@@ -0,0 +1,262 @@
+/**
+ * Mock for the vscode module used in unit tests.
+ */
+
+const EventEmitter = require('events');
```

ESLint

[error] 'EventEmitter' is assigned a value but never used.

(no-unused-vars)